dbt-bigquery/src/dbt/adapters/bigquery/impl.py

```diff
@@ -816,7 +816,7 @@ def get_table_options(
                 if catalog_relation.table_format == constants.ICEBERG_TABLE_FORMAT:
                     opts["table_format"] = f"'{catalog_relation.table_format}'"
                     opts["file_format"] = f"'{catalog_relation.file_format}'"
-                    opts["storage_uri"] = f"'{catalog_relation.external_volume}'"
+                    opts["storage_uri"] = f"'{catalog_relation.storage_uri}'"
```

Comment:
Will this break existing customers or are the semantics unchanged?

Reply:
+1 here --> no one is using this today on dbt-bigquery so it's totally fine